misc/language-tour-guide/src/content/1-basic-types-and-operations/booleans.livemd

```diff
@@ -0,0 +1,91 @@
+# Booleans and nil
+
+## Booleans
+
+Elixir supports `true` and `false` as booleans, and provides logical `and`, `or` and `not` operators to work on them.
+
+Note that `and` and `or` are short-circuit: if the left-hand side determines the result, the right-hand side won't even be evaluated. It's especially important when doing side effects.
+
+The boolean values:
+
+```elixir
+true
+```
+
+```elixir
+false
+```
+
+Are they equal?
+
+```elixir
+true == false
+```
+
+Logical `and` of two true values:
+
+```elixir
+true and true
+```
+
+Short-circuit `and` - try changing `true` to `IO.inspect(true)` to see if it gets evaluated:
```

Comment:
Previously exercises were clearly marked with the bulb icon - here I would either keep it or find another uniform way